clients/drcachesim/tools/invariant_checker.cpp

```diff
@@ -1023,8 +1025,12 @@ invariant_checker_t::check_schedule_data(per_shard_t *global)
     for (auto &keyval : cpu2sched_file) {
         std::sort(keyval.second.begin(), keyval.second.end(),
                   [](const schedule_entry_t &l, const schedule_entry_t &r) {
-                      if (l.timestamp == r.timestamp)
-                          return l.thread < r.thread;
+                      if (l.timestamp == r.timestamp) {
```

Comment:
Hmm so this one wasn't updated before.  Best to make it use the same code (the != sequence) as the other as otherwise it looks like it's trying to be different.  Use the same code: you can use the same lambda, right?  That seems worth the wasted check for cpu.

Reply:
Done.